This patch was not upstreamed to qemu as those should probably be
defined in musl libc.

--- linux-user/syscall.c.orig	2016-12-21 15:42:53.453155713 +0100
+++ linux-user/syscall.c	2016-12-21 15:43:34.995638423 +0100
@@ -114,6 +114,13 @@
 
 #include "qemu.h"
 
+#ifndef F_SHLCK
+#define F_SHLCK 8
+#endif
+#ifndef F_EXLCK
+#define F_EXLCK 4
+#endif
+
 #ifndef CLONE_IO
 #define CLONE_IO                0x80000000      /* Clone io context */
 #endif
